pages/app/add-testimonial.tsx

```diff
@@ -0,0 +1,135 @@
+import {
+  Input,
+  Textarea,
+  Button,
+  Heading,
+  FormControl,
+  FormLabel,
+  useToast,
+} from '@chakra-ui/react';
+import { ChangeEvent, FormEvent, useState } from 'react';
+
+import Section from '@packages/components/Section';
+import { getTestimonialInstance } from '@packages/services/testimonials';
+import { useI18n } from '@packages/features/i18n-context';
+
+interface testimonialProps {
+  name: string;
+  testimonial: string;
+  gitUsername: string;
+}
+
+export default function AddTestimonialPage() {
+  const { t } = useI18n('testimonials');
+  const [testimonial, setTestimonial] = useState<string>('');
+  const [name, setName] = useState<string>('');
+  const [gitUsername, setGitUsername] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const maxInputLength: number = 300;
+  const toast = useToast();
+
+  function validateSubmit() {
+    if (name.length < 5 || testimonial.length < 20) {
+      toast({
+        description: t('toast.inputError'),
+        status: 'error',
+        isClosable: true,
+      });
+      return false;
+    } else return true;
+  }
```

Comment:
Acredito que essa função pode ser simplificada. Perceba que não há necessidade do else:
```suggestion
function validateSubmit() {
    const isInValidInput = name.length < 5 || testimonial.length < 20;
    if (!isInValidInput) return true;

    toast({
      description: t('toast.inputError'),
      status: 'error',
      isClosable: true,
    });

    return false;
  }
```

Fiz um exemplo acima que considero mais simples, ao invés da comparação booleana estar solta no if, eu a dou um nome, isso traz significado para a verificação que estou fazendo. Em seguida, eu já busco fazer o fluxo de check mais simples, perceba como a função fica mais direta e da evidência a parte mais complexa